- util/cutils: Use getexecpath() on OpenBSD for qemu_init_exec_dir()

Index: util/cutils.c
--- util/cutils.c.orig
+++ util/cutils.c
@@ -1091,6 +1091,12 @@ void qemu_init_exec_dir(const char *argv0)
             p = buf;
         }
     }
+#elif defined(__OpenBSD__) && defined(CONFIG_GETEXECPATH)
+    {
+        if (getexecpath(buf, sizeof(buf)) == 0) {
+            p = buf;
+        }
+    }
 #elif defined(__APPLE__)
     {
         char fpath[PATH_MAX];
